bip-0003.md

```diff
@@ -0,0 +1,694 @@
+may be ignored/overruled by general agreement that they have been sufficiently addressed, but clear reasoning must be
+given in such circumstances. Deployed Process BIPs may be modified indefinitely as long as a proposed modification has
+rough consensus per the same criteria.[^living-documents]
+
+#### Closed[^closed]
+
+A BIP that is _not in active use_, may be labeled Closed when its authors have stopped working on it, no longer
+recommend the proposed approach, or no longer pursue adoption of their Complete proposal. The reason for moving the
+proposal to Closed should be recorded in the Changelog section in the same commit that updates the status.
+
+##### Draft ↦ Closed
+
+BIP authors may decide on their own to change their BIP’s status from Draft to Closed. If a Draft BIP stops making
+progress, sees accumulated feedback unaddressed, or otherwise appears stalled for a year, the community may move the BIP
+to Closed unless the authors assert that they intend to continue work within four weeks of being contacted.
+
+##### Complete ↦ Closed
+
+BIPs that had attained the Complete status, i.e., that had been recommended for adoption, may be moved to Closed per the
+authors’ announcement to the Bitcoin Development Mailing List after not being opposed within four weeks. To successfully
+oppose the move, at least one of the opposers must volunteer to become the BIP’s author or shepherd for the BIP to
+remain Complete. A BIP can also be moved to Closed by the community if it has had Complete status for at least one year,
+there is no evidence of it being in active use, and its authors do not object or fail to respond, unless a new author
+volunteers within four weeks.
+
+##### Deployed ↦ Closed
+
+A BIP may evolve from Deployed to Closed when it is no longer in active use. Any community member may initiate this
+Status update by announcing it to the mailing list, and proceed if no objections have been raised for four weeks.
+
+##### Closed ↦ Draft
+
+The Closed status is generally intended to be a final status for BIPs.
+
+If BIP authors decide to make another attempt at a previously Closed BIP, it is generally recommended to create a new
+proposal. (Obviously, the authors may borrow any amount of inspiration or actual text from any prior BIPs as licensing
+permits.) The authors should take special care to address the issues that caused the prior attempt’s abandonment. Even
+if the prior attempt had been assigned a number, the new BIP will generally be assigned a distinct number, unless it is
+obvious that the new attempt directly continues work on the same idea, in which case it may be reasonable to return the
+Closed BIP to Draft status.
+
+### Changelog
+
+To help implementers understand updates to a BIP, any changes after it has reached Complete must be tracked with version,
+date, and description in a Changelog section. The version number is inspired by semantic versioning (MAJOR.MINOR.PATCH).
```

Comment:
Suggest stipulating that entries be ordered by most recent first / descending date (i.e. like in https://keepachangelog.com/)